lib/internal/modules/cjs/loader.js

```diff
@@ -567,7 +569,16 @@ function tryModuleLoad(module, filename) {
 }
 
 Module._resolveFilename = function(request, parent, isMain, options) {
-  if (NativeModule.nonInternalExists(request)) {
+  if (request.search('@nodejs/') === 0) {
```

Comment:
I have a slight preference to use `indexOf` here.

Reply:
Or `startsWith`?